stock_move_entries/model/stock_move_entries.py

```diff
@@ -114,7 +114,7 @@ def _create_account_move_line(self, cr, uid, move, src_account_id,
 
 
```

Comment:
Get rid of one of these two inheritance as they are doing the same think.

Best Regards.